src/index.html

```diff
@@ -21,7 +24,436 @@
+          src="./images/logo.png"
+          alt="MOYO Header Image"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="product-section page__product-section">
+      <article class="card">
+        <a
+          href="#product"
+          class="card__img-link"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__img"
+          />
+        </a>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars-container">
+          <ul class="stars stars--4 card__stars">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </button>
+      </article>
+      <article
```

Comment:
fix ur indents in all the cases
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules
<img width="1061" alt="Знімок екрана 2024-08-13 о 17 32 59" src="https://github.com/user-attachments/assets/7d83722a-0253-4be5-947b-cfc003929ff6">

```suggestion
      </article>
      
      <article
```